src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -1992,6 +2003,30 @@ namespace SurfaceGeometry {
 
 	}
 
+	bool
+	isAzimuthDifferent( Real64 azimuth1, Real64 azimuth2 )
+	{
+		return ( std::abs( azimuth1 - azimuth2 ) > 10.0 );
```

Comment:
@rglazer2 @JasonGlazer We may need to rethink this.  The function name here is a bit misleading since there is a hardwired tolerance.  Later on in SurfaceGeometry a similar set of tests are performed on non-horizontal surfaces but with different tolerances, see lines [1351-1375](https://github.com/rglazer2/EnergyPlus/blob/46e58cc4611071e51066ca1cdc9bd78053c30893/src/EnergyPlus/SurfaceGeometry.cc#L1351-L1375)
Seems the new functions should be used there as well.  That section already has code in place to report an error regardless of DisplayExtraWarnings.  I'm not sure why the tolerances are different in these two places.